src/Subclasses/User.php

```diff
@@ -22,6 +22,14 @@ class User extends ParseUser implements Authenticatable
 {
     use CastsParseProperties;
 
+	/**
+     * @return string
+     */
+    public function getAuthIdentifierName()
+    {
+        return $this->getUsername();
```

Comment:
you know, I saw your return ‘username’ comment and that’s why I switched it.  I guess I didn’t see where you had noted to changed it to objectId.  Sorry for taking a bit to understand.  Appreciate the help!

> On Apr 28, 2016, at 1:34 AM, nicklee1990 <notifications@github.com> wrote:
> 
> In src/Subclasses/User.php <https://github.com/HipsterJazzbo/LaraParse/pull/47#discussion_r61391262>:
> 
> > @@ -22,6 +22,14 @@ class User extends ParseUser implements Authenticatable
> >  {
> >      use CastsParseProperties;
> >  
> > +	/**
> > +     * @return string
> > +     */
> > +    public function getAuthIdentifierName()
> > +    {
> > +        return $this->getUsername();
> return 'username'
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/HipsterJazzbo/LaraParse/pull/47/files/9f75ac069f81826a8e3dab35b8e7b8e5c6ff9626#r61391262>